crates/common/account_manager/src/message_types.rs

```diff
@@ -0,0 +1,22 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum MessageType {
+    Attestation,
+    Blocks,
```

Comment:
```suggestion
    Block,
```

Can this be singular? And also could you change the file name into `message_type.rs`?